src/classes/TDTM_Config_API_Test.cls

```diff
@@ -42,17 +42,17 @@ private class TDTM_Config_API_Test {
     * @description Test that an individual trigger can be disabled in the current code execution context
     */
     @isTest
-    static void test_disable_trigger_in_context() {
+    static void triggersAreDisabledForSpecificContext() {
         //skip the test if Advancement is installed
-        if(ADV_PackageInfo_SVC.useAdv()) return;
+        if (ADV_PackageInfo_SVC.useAdv()) return;
         
         TDTM_DefaultConfig.startTestDataInit();
         Test.startTest();
 
-        Contact con1 = new Contact(LastName='foo');
+        Contact con1 = new Contact(LastName = 'foo');
```

Comment:
Minor, but would you mind changing all the places in the test class that create a new Contact to use `UTIL_UnitTestData_TEST.getContact();`?

For example: `Contact con1 = UTIL_UnitTestData_TEST.getContact();`

Reply:
All set. Thank you, @force2b ! I rarely look at this side of the codebase, so I just learned something new. :)

Reply:
All set. Thanks, @force2b ! I rarely look at this side of the codebase, so I learned something new. :)